server/openslides/users/serializers.py

```diff
@@ -97,6 +103,15 @@ def validate(self, data):
         if "about_me" in data:
             data["about_me"] = validate_html_strict(data["about_me"])
 
+        if "vote_delegated_to" in data:
```

Comment:
Here we are missing two autoupdates. If the id changed from 2 to 3, we need autoupdates for users 2 and 3, since the reverse relation needs to be updated. See https://github.com/OpenSlides/OpenSlides/blob/master/server/openslides/motions/serializers.py#L511 for reference (there we are doing it for motion block). You should be able to test this by either capturing the autoupdate in tests or accessing the element cache and get the data for each user and validate the ids.